StoryboardsAndXibs.md

```diff
@@ -10,18 +10,42 @@ In most cases on both macOS and iOS, it is recommended to avoid storyboards (and
     - [Compose view hierarchies With NSStackViews/UIStackViews](Layout.md#Compose-Hierarchies-with-NSStackViews/UIStackViews).
     - [Use layout anchor constraints](Layout.md#Use-Layout-Anchor-Constraints)
     - [Create the view hierarchy in loadView](ViewControllers.md#Construct-View-Hierarchies-in-loadView)
+- Not supported in VisionOS
 
 ## Exceptions
-- Launch Screens on iOS are required to be specified as storyboards.
 - The main menu bar for a macOS app is required to be specified in a XIB.
 
-## How to convert xib to programmatic initialization
+## How to convert xib/storyboards to programmatic initialization on iOS
+- Remove any reference to loading UIStoryboard pointer [storyboardWithName:bundle:](https://developer.apple.com/documentation/uikit/uistoryboard/1616216-storyboardwithname)
 - Change the UIViewController caller
-    - Instead of passing in [nibName](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621359-init) in particular bundle, pass in nil
-    - Make sure no other storyboard is initializing with [iniWithCoder:](https://developer.apple.com/documentation/foundation/nscoding/1416145-initwithcoder?language=objc)
+    - If the UIViewController was initialized by storyboard, instead of [instantiateViewController(withIdentifier:)](https://developer.apple.com/documentation/uikit/uistoryboard/1616214-instantiateviewcontroller), use [init(nibName:bundle:)](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621359-init) with parameter nil for nibName and bundle.
+    - If the UIViewController was initialized by xibs, instead of passing in [init(nibName:bundle:)](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621359-init) in particular bundle, pass in nil for nibName and bundle.
+    - Make sure no content of storyboard is initializing with [initWithCoder:](https://developer.apple.com/documentation/foundation/nscoding/1416145-initwithcoder?language=objc)
+- For storyboards, make sure segues related functions do NOT get called. 
```

Comment:
The rest of the guidance is saying not to use storyboard, but this is saying "For storyboards", which seems like it's saying there is a case for storyboards? Maybe I'm just misunderstanding the guidance in this step

Reply:
oh i meant these performsegue is only available in storyboard case rather than xib. maybe i can say "if you were using storyboard"